src/dune_rules/coq_rules.ml

```diff
@@ -19,6 +20,13 @@ module Util = struct
         let info = Lib.info t in
         Lib_info.src_dir info)
 
+  let native_paths ts =
```

Comment:
I think this function needs a better name. To me, the name it implies it wants the directories of `.cmx` files, but that's not what it's doing.

Reply:
Indeed, that's used to tell Coq where to look for the -I flags of the coq libraries that are needed, in this case `coq.kernel`. So maybe `coq_native_mllibs_include_paths` ?

Reply:
I see. Something like `public_cmi_dirs` sounds enough to me.

Reply:
Used `coq_nativelib_cmi_dirs` , as we will at some point rename this OCaml lib to `coq.nativelib`, thanks!